documentation/tutorial.md

```diff
@@ -160,93 +168,77 @@ selectors in Crawly shell.
+  ],
   request: %HTTPoison.Request{
     body: "",
     headers: [],
     method: :get,
     options: [],
     params: %{},
-    url: "https://www.homebase.co.uk/our-range/tools"
+    url: "https://books.toscrape.com/"
   },
-  request_url: "https://www.homebase.co.uk/our-range/tools",
+  request_url: "https://books.toscrape.com/",
   status_code: 200
 }
 ```
 
-Using the shell, you can try selecting elements using Floki with the
-response. Let's say that we want to extract all product categories links from the
-page above:
+Now let's start parsing the document and extract the books in it.
 
 ```
-{:ok, document} = Floki.parse_document(response.body)
-document |> Floki.find("section.wrapper") |> Floki.find("div.article-tiles.article-tiles--wide a") |> Floki.attribute("href")
+response = Crawly.fetch("https://books.toscrape.com/")
 
-["/our-range/tools/power-tools", "/our-range/tools/garage-storage",
- "/our-range/tools/hand-tools", "/our-range/tools/tool-storage",
- "/our-range/tools/ladders", "/our-range/tools/safety-equipment-and-workwear",
- "/our-range/tools/work-benches"]
 ```
 
-The result of running the command above is a list of elements which
-contain href attribute of links selected with
-`a.category-block-heading__title` CSS selector. These URLs will be
-used in order to feed Crawly with requests to follow.
-
-In order to find the proper CSS selectors to use, you might find
-useful opening the target page from the shell in your web browser. You
-can use your browser developer tools to inspect the HTML and come up
-with a selector.
-
-Now let's navigate to one of the Homebase product pages and extract
-data from it.
+All books on the page are enclosed in a `.product_pod` class. We can the select the class and pick the information we want from it.
 
 ```
-response = Crawly.fetch("https://www.homebase.co.uk/bosch-universalimpact-800-corded-hammer-drill_p494894")
-
+items =
+  document
+  |> Floki.find(".product_pod")
+  |> Enum.map(fn x ->
+    %{
+      title: Floki.find(x, "h3 a") |> Floki.attribute("title") |> Floki.text(),
+      price: Floki.find(x, ".product_price .price_color") |> Floki.text()
+    }
+  end)
 ```
 
-Extract the `title` with:
+Here we select all products using the class name, iterate through it and pick the title using `Floki.find(x, "h3 a") |> Floki.attribute("title") |> Floki.text()`
+and pick the price using `Floki.find(x, ".product_price .price_color") |> Floki.text()`. The full text of the book name is only contained
+in the title attribute and not in the text node of the `a` tag, hence our use of the title attribute.
 
-```
-{:ok, document} = Floki.parse_document(response.body)
-document |> Floki.find(".page-title h1") |> Floki.text()
-"Bosch UniversalImpact 800 Corded Hammer Drill"
-```
-
-Extract the `SKU` with:
+To handle pagination, we'll tell crawly to visit the next page by extracting the next pages' urls and building new requests from them.
```

Comment:
we'll tell Crawly

Refer to Crawly with capitalized C.

Add in the following note as well:

> Pagination url extraction can be within the same callback, as if there are no pagination HTML nodes found, `next_requests` would be an empty list.

Reply:
done